### DISCUSSION TITLE
how to ignore these errors when exporting project with ILSpy

### DISCUSSION THREAD

**Discussion by xuan2261** [icsharpcode/ILSpy#2502] [Help]
i use ilspy export project but it is very error CS0103. example
![image](https://user-images.githubusercontent.com/17797687/133476165-18c28374-f45a-4889-a70f-b42bb1c8ff4b.png)
![image](https://user-images.githubusercontent.com/17797687/133476195-e65c9a51-4afb-4a64-abbc-37525a15cb2a.png)
Currently there is no option to bypass this error in ILSpy. Anyone help me fix or skip many error?
Thanks you very much.

**Comment by siegfriedpammer** [comment]
Looks like the decompiler cannot find some of the references. Can you please open an issue and provide the binary you tried to decompile? Thanks!
